src/SFA.DAS.EmploymentCheck.Application/Services/EmployerAccount/EmployerAccountService.cs

```diff
@@ -19,25 +19,52 @@ public class EmployerAccountService : IEmployerAccountService
         private readonly IHashingService _hashingService;
         private readonly IAccountsResponseRepository _repository;
         private readonly IEmployerAccountApiClient<EmployerAccountApiConfiguration> _apiClient;
+        private readonly IApiRetryPolicies _apiRetryPolicies;
 
         public EmployerAccountService(
             IHashingService hashingService,
             IAccountsResponseRepository repository,
-            IEmployerAccountApiClient<EmployerAccountApiConfiguration> apiClient
+            IEmployerAccountApiClient<EmployerAccountApiConfiguration> apiClient,
+            IApiRetryPolicies apiRetryPolicies
         )
         {
             _hashingService = hashingService;
             _repository = repository;
             _apiClient = apiClient;
+            _apiRetryPolicies = apiRetryPolicies;
         }
 
         public async Task<EmployerPayeSchemes> GetEmployerPayeSchemes(Data.Models.EmploymentCheck employmentCheck)
         {
+            HttpResponseMessage response = null;
+
             try
             {
+                var policy = await _apiRetryPolicies.GetAll("AccountApiKey");
                 var hashedAccountId = _hashingService.HashValue(employmentCheck.AccountId);
                 var request = new GetAccountPayeSchemesRequest(hashedAccountId);
-                var response = await _apiClient.Get(request);
+                
+                await policy.ExecuteAsync(async () =>
+                {
+                    response = await _apiClient.Get(request);
+                    if (response != null && !response.IsSuccessStatusCode)
+                    {
+                        if (response.StatusCode == HttpStatusCode.Unauthorized)
+                            throw new UnauthorizedAccessException();
+
+                        throw new HttpException(response.StatusCode);
+                    }
+
+                });
+
+                return await ProcessPayeSchemesFromApiResponse(employmentCheck, response);
+            }
+            catch (HttpException)
+            {
+                return await ProcessPayeSchemesFromApiResponse(employmentCheck, response);
+            }
+            catch (UnauthorizedAccessException)
+            {
```

Comment:
A method shouldn't both throw and catch an exception in the same block of code..